docs/faq/README.md

```diff
@@ -0,0 +1,86 @@
+# FAQ
+
+## Q: How to use private image repositories in OpenFunction?
+
+A: OpenFunction uses Shipwright (essentially using Tekton to integrate with Cloud Native Buildpacks) in the build phase to package the user function to the application image.
+
+When using a private image repository, users often choose to access it in an insecure way, which is not yet supported by the Cloud Native Buildpacks.
```

Comment:
When using a private image repository, users often choose to access it in an insecure way
=>
Users often choose to access a private image repository in an insecure way.